templates/postgresql_database.yml.erb

```diff
@@ -4,9 +4,14 @@ development: &default
   encoding: utf8
   host: localhost
   min_messages: warning
-  pool: 2
+  pool: <%%= ENV.fetch("DB_POOL", 2) %>
+  reaping_frequency: <%%= ENV.fetch("DB_REAPING_FREQUENCY", 10) %>
   timeout: 5000
 
 test:
   <<: *default
   database: <%= app_name %>_test
+
+production:
+  url:  <%%= ENV.fetch("DATABASE_URL") %>
```

Comment:
When running this in my development env it chokes because I don't have `DATABASE_URL` set. Suffice it to say - on heroku this *will* be set, and locally it's not needed. A change to `<%%= ENV.fetch("DATABASE_URL", "") %>` fixed this for me.